config/settings.yml

```
@@ -1176,6 +1176,9 @@ vanotify:
         form40_0247_confirmation_email: form40_0247_confirmation_email_template_id
         form40_0247_error_email: form40_0247_error_email_template_id
         form40_10007_error_email: form40_10007_error_email_template_id
+        form_upload_confirmation_email: form_upload_confirmation_template_id
+        form_upload_error_email: form_upload_error_template_id
+        form_upload_received_email: form_upload_received_template_id
```

I don't see these in [parameter store](https://console.amazonaws-us-gov.com/systems-manager/parameters/?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:env_vars%2Fvanotify%2Fservices%2Fva_gov%2Ftemplate_id). I think they need to be there for each env before this will work, right?

---

That's right. I thought I'd merge this first (which will silently return when template_ids are not found) and then add them to the `vsp-infra-application-manifests` repo ([like this PR of mine about a different form](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3325/files)), starting with staging and then moving on to prod if all looks good. Does that plan sound ok?

---

Yeah, as long as things won't break if one of these `...template_id`s are returned (and not the real template id).

Once you pull from the manifest repo, you'll see things have changed there in dev and staging. See the [announcement here](https://dsva.slack.com/archives/C03R5SBELQM/p1738692758724049). Docs will be updated soon, but basically you're not going to need to add any entries to the manifest repo anymore. All you're going to need to do is create a parameter in AWS (with a specific path) and the variable will be created automatically ✨ BUT, this hasn't fully gone into effect, so... the answer to your question is yes, AFAIK, go ahead and add those settings to the manifest repo like you've done in the past.

---

@Thrillberg, without any higher precedent values to override the settings.yml values, those values (e.g., `form_upload_confirmation_template_id`) will be live in all deployed environments. I just want to make sure the "not found" part means it's not included in a list vs nil, because the value will exist

Also I agree with Rebecca, add the settings to the manifest repo and we'll clean them up later.